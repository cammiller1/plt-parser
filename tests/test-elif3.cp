## test that the if, else if, else can go to a sub-branch ##

int x;
int y;
y = 3;
x = 1;
if (x > 10) {
	prints("went to if branch");
}
else if (x == 1) {
	
	if (y == 3) {
		prints("went to the inner if in the elif branch");
	}
}
else if (x == 2) {
	prints("went to second elif branch");
} else {
	prints("went to else branch");
}