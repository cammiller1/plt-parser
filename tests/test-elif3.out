went to the inner if in the elif branch